src/partials/header-content.hbs

```diff
@@ -21,7 +21,19 @@
                     </div>
                     <a class="menu-item" href="/en-gb/plentydevtool/main/plentydevtool-introduction.html">plentyDevTool</a>
                     <a class="menu-item" href="https://knowledge.plentymarkets.com/en" target="_blank">User manual</a>
-                </div>
+                    <div class="dropdown menu-item">
```

Comment:
If we want to use only one header-content.hbs, we have to add a conditional that the element doesn't show up on developers.plentymarkets.com.

Reply:
check if its correct!

Reply:
It looks right to me at least, yeah.